packages/client/test/unit/MessageFactory.test.ts

```diff
@@ -116,7 +116,7 @@ describe('MessageFactory', () => {
         expect(msg.groupKeyId).toBe(GROUP_KEY.id)
         expect(msg.newGroupKey).toMatchObject({
             groupKeyId: nextGroupKey.id,
-            encryptedGroupKeyHex: expect.any(String)
+            data: expect.any(Buffer)
```

Comment:
Should it be exactly `Uint8Array`?

```suggestion
            data: expect.any(Uint8Array)
```